.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -47,17 +47,20 @@ env:
 !{{ common.concurrency(build_environment) }}
 
 jobs:
-!{{ common.ciflow_should_run_job(ciflow_config) }}
 {% block build +%}
   build:
     runs-on: linux.2xlarge
-    needs: [!{{ ciflow_config.root_job_name }}]
     timeout-minutes: !{{ common.timeout_minutes }}
+    if: !{{ ciflow_config.root_job_condition }}
     env:
       JOB_BASE_NAME: !{{ build_environment }}-build
+      IS_PROBOT_TRIGGER_EVENT: ${{ (github.event.action == '!{{ ciflow_config.trigger_action }}') && (github.event.assigneed.login == '!{{ ciflow_config.trigger_actor }}') }}
+      LABEL_CONDITIONS: ${{ !{{ ciflow_config.label_conditions }} }}
     outputs:
       docker_image: ${{ steps.calculate-tag.outputs.docker_image }}
     steps:
+      - name: print labels
+        run: echo "${LABELS}"
```

Comment:
Rename PR_LABELS variable above to LABELS (otherwise I think this is just empty)?

Though I believe there may be other uses of PR_LABELS in the code so renaming those would be advised as well

Reply:
I just renamed everything to PR_LABELS and got rid of a few more LABELS after realizing it gets initialized in linux_ci_workflow.